lib/ipam/ipam_test.go

```diff
@@ -513,6 +514,9 @@ var _ = testutils.E2eDatastoreDescribe("IPAM tests", testutils.DatastoreEtcdV3,
 				Hostname:  host,
 				IPv4Pools: []cnet.IPNet{fromPool},
 			}
+			if strings.Contains(host, "block-limit") {
```

Comment:
Couldn't we just add a new argument to this?

Might be useful to be able to test block limits other than 1 or 0.